### ISSUE TITLE
Links are missing discernible text

### ISSUE THREAD

**Issue by sarahocarroll** [socarra/automation_assessment#6]
- Run Axe tests on https://automationintesting.online/
- Reports that there are 3 links on the page missing text
-> facebook icon
-> instagram icon
-> twitter icon

<img width="3777" height="2160" alt="Image" src="https://github.com/user-attachments/assets/5b377bcc-112f-49f7-a737-3214158bb08c" />
